src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -299,7 +313,7 @@ public async Task FlushAsync(bool force = true)
                     u.Value.ExistInInner = true;
                 }
 
-                await this.Inner.SaveChangesAsync(unspent.Select(u => u.Value.UnspentOutputs).ToArray(), null, this.innerBlockHash, this.blockHash, this.cachedRewindDataList).ConfigureAwait(false);
+                await this.Inner.SaveChangesAsync(unspent.Select(u => u.Value.UnspentOutputs).ToArray(), null, this.innerBlockHash, this.blockHash, 0, this.cachedRewindDataList.Select(c => c.rewindData).ToList()).ConfigureAwait(false);
```

Comment:
Still not sure how the height will be found?

Reply:
`Inner` coin view is a dbreeze storage. All rewind data is stored in dbreeze with key=height. So for dbreeze we don't need to pass height in, I only need it in cached coin view. But because they both implement `ICoinView` I have to pass some value in.